networking/interconnect/variables.tf

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "router_name" {
```

Comment:
As a general rule, don't use many variables but bundle up variables related to a logical resource together. For example, you might want to put name and description under a single `router` variable of type `object`.

Please also always provide sensible defaults for variables that can be omitted. One of the main goals of our modules is to make code leaner and more legible for the most common use cases.